packages/gatsby-theme-docs/gatsby-node.mjs

```diff
@@ -348,6 +349,9 @@ export const onCreateNode = async (
         ? Number(node.frontmatter.timeToRead)
         : 0,
       tableOfContents: await generateToC(nodeBodyAst),
+      courseId: node.frontmatter.courseId
+        ? Number(node.frontmatter.courseId)
+        : 0, // TODO: understand how to make an Number field nullable
```

Comment:
Answer: remove the exclamation mark in the schema.   https://graphql.org/learn/schema/#object-types-and-fields